apps/web/src/app/(backButton)/schedule/register/page.tsx

```diff
@@ -1,3 +1,94 @@
-export default function ScheduleRegister() {
-    return <div className="grid items-center justify-items-center min-h-screen">schedule register</div>
+'use client'
+
+import React, { useState } from 'react'
+import { IMakeNewScheduleRequest } from '@fienmee/types'
+
+export default function ScheduleRegister({ title, startDate, endDate, description }: IMakeNewScheduleRequest) {
+    const [isAllDay, setIsAllDay] = useState<boolean>(false)
```

Comment:
```suggestion
export default function ScheduleRegister({ params }: { params: Promise<IMakeNewScheduleRequest>}) {
    const [isAllDay, setIsAllDay] = useState<boolean>(false)
    const { name, startDate, endDate, description } = use(params)
```

이런식으로 수정해야 build 오류는 발생하지 않을 거에요.

https://nextjs.org/docs/app/api-reference/file-conventions/page

해당 문서 한번 확인하시면 좋을 것 같아요. 의도하신 바가 searchParam으로 받는 것인지, param으로 넘겨주는 지에 따라 수정이 필요할 것 같습니다.

Reply:
제가 next를 이해를 잘 못하고있었네요 감사합니다 :)